### ISSUE TITLE
[Bug]: Property type of tokens brought by an imported map is changed to Basic

### ISSUE THREAD

**Issue by fw** [RPTools/maptool#4969]
### Describe the Bug

The property type of tokens brought by an imported map has "Basic" rather than the specified type before importing.

### To Reproduce

1. Add a property type named "Test"
2. Add a token then set the property type to "Test"
3. Export a map with that token
4. Import the map exported in step 3
5. The token on the imported map has the "Basic" property type, not the "Test"

### Expected Behaviour

Tokens have the same property type before exporting.

### MapTool Info

1.15.2

### Desktop

Windows 11 Pro

### Additional Context

I tested in 1.13.0, 1.14.3, 1.15.0, and 1.15.2.
1.13.0 does correctly; others occur the issue.

**Comment by FullBleed** [comment]
Imported maps don't save token types... only campaign files.

The same will happen if you save out a token and import it.

**Comment by Baaaaaz** [comment]
Similarly reported here: https://github.com/RPTools/maptool/issues/4607 which has some additional comments.

**Comment by FullBleed** [comment]
Yeah... should clarify that, as far as I know, it is expected that the token type will become "Basic" if you're importing it into a campaign that does not have the token's original type.  The token is not a carrier for the token type to the campaign... and the map import probably isn't either (for the same reasons).

@fw When you say that 1.13.0 does it correctly are you saying that importing the map in that version carried a New token type into another campaign?  Or that it does it correctly only when the existing campaign already had the type?

I know this subject has come up many times (token's being a carrier of their type)... and I believe that there were concerns that importing types with the same name, but actually different, could cause trouble.  I don't know if there was an "answerr" to that concern...

Personally, I think:

1)  If the type exists in the target campaign it should compare to the token's existing type.
     a)  If it aligns (same properties), let it pass through and keep its type (which is what it appears to have originally done in 1.13.0.)
     b)  If it's different, rename it to something that the user will easily see is an issue that needs addressing (i.e. originalName+X).  Naming it "Basic" is not a good idea.

2)  If the token type does not exist, it would be nice if it "created" the new type in the campaign.

**Comment by fw** [comment]
When the campaign importing to already has the same token type, it does correctly.
